packages/crud/theme/lumo/vaadin-crud-styles.js

```diff
@@ -103,17 +103,22 @@ registerStyles(
         position: relative;
       }
 
-      [part='editor']:focus::before {
+      [part='editor']:focus [part='footer']::before {
+        content: '';
         position: absolute;
         inset: 0;
-        content: '';
-        box-shadow: inset 0 0 0 var(--_focus-ring-width) var(--_focus-ring-color);
+        border: solid var(--_focus-ring-width) var(--_focus-ring-color);
+        border-top: 0;
```

Comment:
The focus ring solution here causes some issues on Windows, note that the right border of the focus ring is not visible, and that the footer has a right focus ring border on the left of the scrollbar:
![Bildschirmfoto 2024-03-26 um 10 06 15](https://github.com/vaadin/web-components/assets/357820/4d301b80-3a0b-4ff6-af1d-3ac1cdae97d3)

It was looking like this before:
![Bildschirmfoto 2024-03-26 um 10 07 16](https://github.com/vaadin/web-components/assets/357820/222f4b83-e593-4ecc-83ec-d898d042504d)

Reply:
It's reproducible on Mac OS too when the vertical scrollbar is visible.

<img width="1532" alt="image" src="https://github.com/vaadin/web-components/assets/5039436/030c5e57-c3fb-464e-9851-80c88ff347e8">